### ISSUE TITLE
[Test Failure] Che7 Java Selenium test times out during project selection

### ISSUE THREAD

**Issue by ScrewTSW** [redhat-developer/rh-che#1511]
Parent: https://github.com/redhat-developer/che-functional-tests/issues/476

![image](https://user-images.githubusercontent.com/16451875/62044616-ad152180-b203-11e9-95de-6bffc2a90dd7.png)

The test time out trying to select a demo project for the workspace. Should disappear with usage of devfiles and updated dashboard (projects are now being assigned automatically based on devfile)

**Comment by Katka92** [comment]
Seen again on 1a 2x , 1b 2x

**Comment by Katka92** [comment]
Seen on 1a and 1b clusters.

**Comment by ScrewTSW** [comment]
seen today on 1a x4 and 1b x2

**Comment by Katka92** [comment]
Seen on 1a 9x and 1b 8x

**Comment by Katka92** [comment]
Seen on 1a 4x, 1b 5x

**Comment by Katka92** [comment]
Seen on 1a 3x, 1b 4x

**Comment by Katka92** [comment]
Seen on 1a 4x, 1b 4x.

**Comment by Katka92** [comment]
Seen on 1a 6x, 1b 5x.

**Comment by ibuziuk** [comment]
@Katka92 could you confirm that this is the test issue and workspace creation / startup works just fine manually?

**Comment by Katka92** [comment]
@ibuziuk I usually verify that few times a weak, I tested that again now on 1a and 1b and project selection and workspace start up works correctly.

**Comment by ibuziuk** [comment]
ok, looks like a test issue, looking forward for stabilization and switching to the typescript tests

**Comment by Katka92** [comment]
Seen again on 1a 14x, 1b 14x.

**Comment by Katka92** [comment]
Seen again on cluster 2 1x, 1a 6x, 1b 5x.

**Comment by Katka92** [comment]
Seen on cluster 2 3x, 1a 5x, 1b 5x.

**Comment by Katka92** [comment]
Seen on cluster 2 1x, 1a 1x, 1b 1x.

**Comment by ScrewTSW** [comment]
appeared yesterday and today on 2,1a and 1b

**Comment by ScrewTSW** [comment]
seen today again:
`starter-us-east-2` x2
`starter-us-east-1a` x5
`starter-us-east-1b` x4

**Comment by ScrewTSW** [comment]
seen today at:
`starter-us-east-2` x 15
`starter-us-east-1a` x 16
`starter-us-east-1b` x 17

**Comment by ScrewTSW** [comment]
Happened to day again:
`starter-us-east-2` x 4
`starter-us-east-1a` x 5
`starter-us-east-1b` x 5

**Comment by ScrewTSW** [comment]
> Worth noting that I think this is a rc-1.1 specific issue, since the dashboard has changed for Che 7.0.0 (devfiles include projects, you don't select one anymore).

yes, but I'm still obliged to track it until we finally update prod to che7 GA :)

**Comment by ScrewTSW** [comment]
Happened to day again:
`starter-us-east-2` x 4
`starter-us-east-1a` x 6
`starter-us-east-1b` x 6

**Comment by ScrewTSW** [comment]
Happened again today:
`starter-us-east-2` x 5
`starter-us-east-1a` x 5
`starter-us-east-1b` x 5

**Comment by ScrewTSW** [comment]
Happened again today:
`starter-us-east-2` x 6
`starter-us-east-1a` x 6
`starter-us-east-1b` x 6

**Comment by ScrewTSW** [comment]
Happened over the weekend and today as well:
`starter-us-east-2` x 14
`starter-us-east-1a` x 17
`starter-us-east-1b` x 16

**Comment by ScrewTSW** [comment]
Happened again today:
`starter-us-east-2` x 6
`starter-us-east-1a` x 6
`starter-us-east-1b` x 5

**Comment by ScrewTSW** [comment]
Happened again today:
`starter-us-east-2` x 6
`starter-us-east-1a` x 6
`starter-us-east-1b` x 6

**Comment by ScrewTSW** [comment]
Happened again today:
`starter-us-east-2` x 6
`starter-us-east-1a` x 6
`starter-us-east-1b` x 6

**Comment by Katka92** [comment]
Happened again in last three days:
starter-us-east-2 15x
starter-us-east-1a 15x
starter-us-east-1b 15x

**Comment by Katka92** [comment]
Happened again yesterday and today:
starter-us-east-2, starter-us-east-1a, starter-us-east-1b

**Comment by Katka92** [comment]
Happened again today:
starter-us-east-2, starter-us-east-1a, starter-us-east-1b
